css-text-3/Overview.bs

```diff
@@ -1621,10 +1615,20 @@ Text Processing</h3>
           white-space-processing-047.xht
         </wpt>
       <li>If there remains any sequence of <a>white space</a>,
-        [=other space separators=],
-	and/or [=preserved=] [=tabs=]
+        and/or [=other space separators=],
         at the end of a line:
         <ul>
+          <li>If 'white-space' is set to ''white-space/normal'', ''white-space/nowrap'', or ''white-space/pre-line'',
```

Comment:
I think the condition is 1) when trailing collapsible spaces are removed, or 2) when trailing collapsible spaces hang. I understand we need some more discussion when to hang for `pre-wrap` but is this intention agreeable?